src/components/cylc/tree/TreeItem.vue

```diff
@@ -349,16 +366,13 @@ export default {
       // returns child nodes folling the family tree and following sort order
       return getNodeChildren(this.node, this.cyclePointsOrderDesc)
     },
-    /** Get the node indentation in units of em. */
+    /** Get the node indentation in units of px. */
     nodeIndentation () {
-      return this.depth * NODE_DEPTH_OFFSET
-      // Note: this should actually depend on the expand/collapse icon size, but
-      // fortuitously, the vuetify default icon size is 24px which is 1.5em for
-      // a font-size of 16px
+      return this.depth * this.indent
     },
     nodeStyle () {
       return {
-        'padding-left': `${this.node.type === 'job-details' ? 0 : this.nodeIndentation}em`
+        'padding-left': `${this.node.type === 'job-details' ? 0 : this.nodeIndentation}px`
```

Comment:
These need to be non-relative otherwise they are dependent on font size which can be changed by the user, leading to incorrect spacing. In Vuetify, spacing is not relative: https://vuetifyjs.com/en/styles/spacing/#how-it-works

Reply:
>  otherwise they are dependent on font size which can be changed by the user

But this **should** be dependent on the font size set by the user right?

Reply:
Vuetify uses a mixture of absolute and relative units pending this slow-moving issue - https://github.com/vuetifyjs/vuetify/issues/3401

tldr; absolute units are fragile and considered bad for reasons including responsive design and accessibility so we've tried to keep them out of our code. Either `em` or `rem` should suffice for most purposes.

Reply:
Ah ok, the only thing that changes with font-size is the task/job icon size, so that means the arrow/triangle thingy should scale with font-size too. But the indent doesn't depend on font-size.

> absolute units are fragile and considered bad for reasons including responsive design and accessibility

Sorry but I do not think there is a clear consensus on this. See https://stackoverflow.com/a/609561/3217306. While it makes sense to use `em`/`rem` for font sizes, or other relative units such as `vw`/`vh` for certain things that depend on responsive design, for the indentation of the tree view I see no particular reason this should scale with font size.

Reply:
* The indentation is based on the expand/collapse icon size.
* The v-icon default size is `1em` [apparently](https://vuetifyjs.com/en/api/v-icon/#sass)
* Therefore the indent size should be in em!

Reply:
`v-btn` doesn't scale with root font size

Reply:
Ok, will address later.